### DISCUSSION TITLE
Autoloader: auto-load all elements matching .customClass

### DISCUSSION THREAD

**Discussion by deining** [markmap/markmap#159] [Q&A] [answered]
Autoloader works great by loading all elements matching `.markmap`. Is there any way to let autoloader (additionally) load all elements matching `.my-custom-class`? I searched the docs, but didn't find anything. Any help is appreciated.

**Comment by gera2ld** [comment, marked as answer]
You can enable the manual mode and pass the elements directly:

```html
<script>
window.markmap = {
  autoLoader: {
    manual: true,
  },
};
document.addEventListener("DOMContentLoaded", () => {
  document
    .querySelectorAll(".markmap-wrapper")
    .forEach(window.markmap.autoLoader.render);
});
</script>
<script src="https://cdn.jsdelivr.net/npm/markmap-autoloader"></script>
```

**Reply by deining** [reply]
Great, thank you!
